examples/aws/cloudformation/ent.yaml

```diff
@@ -478,7 +492,7 @@ Resources:
     CreationPolicy:
       ResourceSignal:
         Count: 2
-        Timeout: PT20M
```

Comment:
We might want to consider running https://gravitational.com/teleport/docs/admin-guide/#troubleshooting and adding a health check to check is the system is ready using http://127.0.0.1:3000/healthz

Reply:
There's actually some stuff in the startup scripts for Cloudformation specifically which does this - basically a command which sends the "OK I'm working" flag to Cloudformation at the end of the startup process. See https://github.com/gravitational/teleport/blob/master/examples/aws/cloudformation/ent.yaml#L229-L255